samples/bluetooth/nrf_dm/src/main.c

```diff
@@ -104,7 +109,18 @@ static bool data_cb(struct bt_data *data, void *user_data)
 			bt_addr_le_copy(&req.bt_addr, user_data);
 			req.role = DM_ROLE_INITIATOR;
 			req.ranging_mode = peer_ranging_mode_get();
-			req.rng_seed = sys_le32_to_cpu(recv_mfg_data->rng_seed);
+			/* We need to make sure that we only initiate a ranging to a single peer.
+			 * A scan response that is received by this device can be received by
+			 * multiple other devices which can all start a ranging at the same time
+			 * as a consequence. To prevent this, we need to make sure that we set a
+			 * per-peer random as the random seed. This helps the ranging library to
+			 * avoid interference from other devices trying to range at the same time.
+			 *
+			 * This means that the initiator and the reflector need to set the same value
+			 * for the random seed.
+			 */
+			req.rng_seed =
```

Comment:
This could still cause multiple devices to initiate a ranging right? The difference is that it shall only be successful with one of them. 

Did you consider / would it be possible to send the addr of the scanner in the scan response, and then just initiate the ranging if this addr matches your own?

Reply:
Ok, thanks for explaining.